runelite-client/src/main/java/net/runelite/client/plugins/barrows/BarrowsConfig.java

```diff
@@ -54,4 +54,14 @@ default boolean showBrotherLoc()
 	{
 		return true;
 	}
+
+	@ConfigItem(
+		keyName = "replaceOverlay",
+		name = "Replace Brothers slain overlay",
+		description = "Configures whether to replace the slain brothers overlay with our own"
+	)
+	default boolean replaceOverlay()
```

Comment:
This should probably not be an option, I don't think we add options for replacing the overlays or not anywhere else